CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 * Update `component` from `btmobilesdk` to `braintreeclientsdk` for PayPal's analytics service (FPTI)
 * BraintreeCore
   * Fix bug where `type` was always returned as `Unknown` in `fetchPaymentMethodNonces` (fixes #1099)
+* Send `correlation_id`, when possible, in PayPal analytic events
```

Comment:
Do we want to group all of the FPTI entries together? I had put the other ones above since it applied to all modules not just core - but I don't feel super strongly where they live. Maybe a `Analytics Changes` header or something similar makes sense?

Reply:
Done!